### ISSUE TITLE
Selenium 4.3.0 has deprecated `find_element_by*()` methods

### ISSUE THREAD

**Issue by firasm** [altair-viz/altair_saver#104]
Hi,

Just filing this issue - doing 

`chart.save()` results in this error:

```
File "scripts/visualize_questions.py", line 61, in <module>
[11](https://github.com/open-resources/physics_bank/runs/7099958015?check_suite_focus=true#step:6:12)
    chart.save('images/topics.png',webdriver='firefox',scale_factor=2)
[12](https://github.com/open-resources/physics_bank/runs/7099958015?check_suite_focus=true#step:6:13)
  File "/opt/hostedtoolcache/Python/3.8.13/x64/lib/python3.8/site-packages/altair/vegalite/v4/api.py", line 488, in save
[13](https://github.com/open-resources/physics_bank/runs/7099958015?check_suite_focus=true#step:6:14)
    result = save(**kwds)
[14](https://github.com/open-resources/physics_bank/runs/7099958015?check_suite_focus=true#step:6:15)
  File "/opt/hostedtoolcache/Python/3.8.13/x64/lib/python3.8/site-packages/altair/utils/save.py", line 116, in save
[15](https://github.com/open-resources/physics_bank/runs/7099958015?check_suite_focus=true#step:6:16)
    mimebundle = spec_to_mimebundle(
[16](https://github.com/open-resources/physics_bank/runs/7099958015?check_suite_focus=true#step:6:17)
  File "/opt/hostedtoolcache/Python/3.8.13/x64/lib/python3.8/site-packages/altair/utils/mimebundle.py", line 60, in spec_to_mimebundle
[17](https://github.com/open-resources/physics_bank/runs/7099958015?check_suite_focus=true#step:6:18)
    return altair_saver.render(spec, format, mode=mode, **kwargs)
[18](https://github.com/open-resources/physics_bank/runs/7099958015?check_suite_focus=true#step:6:19)
  File "/opt/hostedtoolcache/Python/3.8.13/x64/lib/python3.8/site-packages/altair_saver/_core.py", line 257, in render
[19](https://github.com/open-resources/physics_bank/runs/7099958015?check_suite_focus=true#step:6:20)
    mimebundle.update(saver.mimebundle(fmt))
[20](https://github.com/open-resources/physics_bank/runs/7099958015?check_suite_focus=true#step:6:21)
  File "/opt/hostedtoolcache/Python/3.8.13/x64/lib/python3.8/site-packages/altair_saver/savers/_saver.py", line 90, in mimebundle
[21](https://github.com/open-resources/physics_bank/runs/7099958015?check_suite_focus=true#step:6:22)
    bundle[mimetype] = self._serialize(fmt, "mimebundle")
[22](https://github.com/open-resources/physics_bank/runs/7099958015?check_suite_focus=true#step:6:23)
  File "/opt/hostedtoolcache/Python/3.8.13/x64/lib/python3.8/site-packages/altair_saver/savers/_selenium.py", line 284, in _serialize
[23](https://github.com/open-resources/physics_bank/runs/7099958015?check_suite_focus=true#step:6:24)
    out = self._extract(fmt)
[24](https://github.com/open-resources/physics_bank/runs/7099958015?check_suite_focus=true#step:6:25)
  File "/opt/hostedtoolcache/Python/3.8.13/x64/lib/python3.8/site-packages/altair_saver/savers/_selenium.py", line 267, in _extract
[25](https://github.com/open-resources/physics_bank/runs/7099958015?check_suite_focus=true#step:6:26)
    driver.find_element_by_id("vis")
[26](https://github.com/open-resources/physics_bank/runs/7099958015?check_suite_focus=true#step:6:27)
AttributeError: 'WebDriver' object has no attribute 'find_element_by_id'
[27](https://github.com/open-resources/physics_bank/runs/7099958015?check_suite_focus=true#step:6:28)
WARNING:tornado.access:404 GET /favicon.ico (127.0.0.1) 0.41ms
```

The important bit is:

```
AttributeError: 'WebDriver' object has no attribute 'find_element_by_id'
```

This seems to have shown up on [StackOverflow as well](https://stackoverflow.com/questions/72773206/selenium-python-attributeerror-webdriver-object-has-no-attribute-find-el) and the fix seems relatively simple. Let me see if I can submit a PR with the fix.

**Comment by palewire** [comment]
Yup. I got this too.

```python
File ~/.local/share/virtualenvs/news-homepages-Qlfa7zLV/lib/python3.9/site-packages/altair_saver/savers/_selenium.py:267, in SeleniumSaver._extract(self, fmt)
    265 driver.get(url)
    266 try:
--> 267     driver.find_element_by_id("vis")
    268 except NoSuchElementException:
    269     raise RuntimeError(f"Could not load {url}")

AttributeError: 'WebDriver' object has no attribute 'find_element_by_id'
```

**Comment by palewire** [comment]
This error is now surfacing [in Altair's unittests](https://github.com/altair-viz/altair/runs/7121705994?check_suite_focus=true).

**Comment by mcp292** [comment]
Workaround:
```sh
pip install selenium==4.2.0
```

**Comment by rencryptofish** [comment]
> Workaround:
> 
> ```shell
> pip install selenium==4.2.0
> ```

thanks this fixed the issue with `altair-saver = "==0.5.0"`

**Comment by saiwing-yeung** [comment]
@soilstack Have you specified to use the `conda-forge` channel?
https://anaconda.org/conda-forge/selenium

**Comment by joelostblom** [comment]
Since Altair 5.2, the functionality of Altair Saver is now available in Altair via the [vl-convert](https://github.com/vega/vl-convert) package. Most of the functionality has been available since 5.0, and [the main addition in 5.2 was PDF export](https://github.com/altair-viz/altair/pull/3244). See [the docs on how to save charts](https://altair-viz.github.io/user_guide/saving_charts.html#png-svg-and-pdf-format) for more details

We are going to archive this repo, so I'm closing all the open issues and PRs before doing so. Try out the new options for saving charts mentioned above and if you run into issues, please open an issue directly in the altair or vl-convert repo.
